diff --git a/dist/lib/bundler/config.cjs.js b/dist/lib/bundler/config.cjs.js
index 3fe00ad9ac51582e1e5648936f9486bcbaa5621d..871b69e563b5e4e8a11f1aec57a8a0b03779932c 100644
--- a/dist/lib/bundler/config.cjs.js
+++ b/dist/lib/bundler/config.cjs.js
@@ -300,16 +300,20 @@ async function createConfig(paths, options) {
       }
     },
     plugins,
-    ...options.moduleFederationRemote && {
-      // TODO: remove this warning skipping as soon as the corresponding bundler limitation
-      // described in issue https://github.com/web-infra-dev/rspack/issues/13635 is fixed
-      // when PR: https://github.com/web-infra-dev/rspack/pull/13636 is merged.
-      ignoreWarnings: [
-        {
-          message: /No version specified and unable to automatically determine one\. No version in description file/
-        }
-      ]
-    }
+    ignoreWarnings: [
+      {
+        module: /@protobufjs\/inquire/,
+        message: /Critical dependency: the request of a dependency is an expression/,
+      },
+      ...(options.moduleFederationRemote
+        ? [
+            {
+              message:
+                /No version specified and unable to automatically determine one\. No version in description file/,
+            },
+          ]
+        : []),
+    ],
   };
 }
 
